crates/typst-library/src/meta/reference.rs

```diff
@@ -189,8 +190,12 @@ impl Show for RefElem {
                     )
                 })
                 .hint(eco_format!(
-                    "you can enable heading numbering with `#set {}(numbering: \"1.\")`",
-                    elem.func().name()
+                    "you can enable {0} numbering with `#set {0}(numbering: \"1.\")`",
```

Comment:
I think the first `{}` should unconditionally use the name. "You can set equation numbering" makes more sense than "You can set math.equation numbering"

Reply:
Yeah I was just changing it!